radixdlt/babylon-wallet-android
# [ABW-3605] - Fix home cards showing when restoring wallet

[Jira Ticket](https://radixdlt.atlassian.net/browse/ABW-3605)

## Description
This PR fixes issue with home cards showing when restoring wallet.


## How to test

1. Uninstall Radix Wallet app
2. Click a valid deep link
3. Install the app
4. Restore wallet
5. Verify that home cards are not showing on the home screen
